assignment.js

```diff
@@ -65,6 +69,9 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+let numbers = [57, 13, 4, 38]
+
+console.log(Math.max(...numbers))
 
```

Comment:
@laraann88 Nice work here using an advanced es6 Spread syntax `...numbers`. Also thanks for teaching me something new `Math.max()`. I never used it before but it gets the job done and will give you back the largest number. Good Job